### ISSUE TITLE
Not Secure label on http pages collides with the URL

### ISSUE THREAD

**Issue by lokesh-krishna** [zen-browser/desktop#3645]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Visit any http site to get the Not Secure label on the URL. The label sits flush with the URL instead of having some padding. 
![image](https://github.com/user-attachments/assets/758f3c08-932c-4c9d-a048-05de875ab852)

Firefox doesn't use a label, only an icon and is free from this issue.

![image](https://github.com/user-attachments/assets/daa8d730-a911-40cb-82ab-11c0dfcd754d)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0

### What platform are you seeing the problem on?

Linux

**Comment by JSGRD22** [comment]
I agree that this should be replaced with a single icon, especially in respect to space constraints in the single toolbar mode.

**Comment by lokesh-krishna** [comment]
Issue is still present in version 1.6b.
